tests/automatic_tags_der.rs

```diff
@@ -0,0 +1,36 @@
+pub use asn1rs::prelude::*;
+
+asn_to_rust!(
+    r"World-Schema DEFINITIONS AUTOMATIC TAGS ::=
+BEGIN
+  Rocket ::= SEQUENCE
+  {
+     range     INTEGER, -- huge (see a special directive above)
+     name      UTF8String (SIZE(1..16)),
+     message   UTF8String OPTIONAL,
+     fuel      ENUMERATED {solid, liquid, gas},
+     speed     CHOICE
+     {
+        mph    INTEGER,
+        kmph   INTEGER
+     }  OPTIONAL,
+     payload   SEQUENCE OF UTF8String
+  }
```

Comment:
As much as I like this example, for legal reasons I cannot accept their example in this repo